src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -277,64 +294,66 @@ class WorkspaceMembersPage extends React.Component {
 
         return (
             <ScreenWrapper style={[styles.defaultModalContainer]}>
-                <HeaderWithCloseButton
-                    title={this.props.translate('workspace.common.members')}
-                    subtitle={policyName}
-                    onCloseButtonPress={() => Navigation.dismissModal()}
-                    onBackButtonPress={() => Navigation.navigate(ROUTES.getWorkspaceInitialRoute(policyID))}
-                    shouldShowGetAssistanceButton
-                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
-                    shouldShowBackButton
-                />
-                <ConfirmModal
-                    danger
-                    title={this.props.translate('workspace.people.removeMembersTitle')}
-                    isVisible={this.state.isRemoveMembersConfirmModalVisible}
-                    onConfirm={() => this.removeUsers()}
-                    onCancel={this.hideConfirmModal}
-                    prompt={this.props.translate('workspace.people.removeMembersPrompt')}
-                    confirmText={this.props.translate('common.remove')}
-                    cancelText={this.props.translate('common.cancel')}
-                />
-                <View style={[styles.pageWrapper, styles.flex1]}>
-                    <View style={[styles.w100, styles.flexRow]}>
-                        <Button
-                            medium
-                            success
-                            text={this.props.translate('common.invite')}
-                            onPress={this.inviteUser}
-                        />
-                        <Button
-                            medium
-                            danger
-                            style={[styles.ml2]}
-                            isDisabled={this.state.selectedEmployees.length === 0}
-                            text={this.props.translate('common.remove')}
-                            onPress={this.askForConfirmationToRemove}
-                        />
-                    </View>
-                    <View style={[styles.w100, styles.mt4, styles.flex1]}>
-                        <View style={[styles.peopleRow]}>
-                            <View style={[styles.peopleRowCell]}>
-                                <Checkbox
-                                    isChecked={this.state.selectedEmployees.length === removableMembers.length && removableMembers.length !== 0}
-                                    onPress={() => this.toggleAllUsers()}
-                                />
-                            </View>
-                            <View style={[styles.peopleRowCell, styles.flex1]}>
-                                <Text style={[styles.textStrong, styles.ph5]}>
-                                    {this.props.translate('workspace.people.selectAll')}
-                                </Text>
+                <FullPageNotFoundView shouldShow={_.isEmpty(this.props.policy)}>
```

Comment:
NAB, any time I have changes like this, I like to leave a comment in the OP saying to view the diff without whitespace spaces so that the diff is less confusing, since all you really did here was wrap everything in a `FullPageNotFoundView`.

You can view diffs without whitespace changes by appending `w=1` to the end of the PR files link, i.e.

https://github.com/Expensify/App/pull/10668/files?w=1